docs/pages/guides/configuring-js-engines.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: Configuring JS Engines
+sidebar_title: Configuring JS Engines
+description: A guide on configuring JS Engines for both Android and iOS in an Expo project.
+---
+
+import { Collapsible } from '~/ui/components/Collapsible';
+import { Terminal } from '~/ui/components/Snippet';
+
+JavaScript engines are responsible for executing your application code and providing various features like memory management, optimization, and error handling. Expo projects, by default, use [Hermes](https://hermesengine.dev/) as the JavaScript engine (in SDK 47 and lower, the default was JSC). It is also possible to switch to other engines like JSC or V8 for both Android and iOS platforms, but not for web because the JavaScript engine is included in the web browser.
+
+### Configuring the `jsEngine` through Expo config
+
+We recommend Hermes because it is purpose built and optimized for React Native apps, and it has the best debugging experience. If you are familiar with the tradeoffs of different JavaScript engines and would like to change away from Hermes, the `jsEngine` field inside **app.json** allows you to specify the JavaScript engine for your app.
+
+The default value is `hermes`. If you would like to use [JavaScriptCore](https://developer.apple.com/documentation/javascriptcore) instead, set the `jsEngine` field in your **app.json**:
```

Comment:
```suggestion
We recommend Hermes because it is purpose built and optimized for React Native apps, and it has the best debugging experience. If you are familiar with the tradeoffs of different JavaScript engines and would like to change away from Hermes, the [`jsEngine`](/versions/latest/config/app/#jsengine) field inside [Expo config](/workflow/configuration/) allows you to specify the JavaScript engine for your app. The default value is `hermes`. 

If you want to use JSC instead, set the `jsEngine` field in the Expo config:
```